styleguide/css/main.scss

```diff
@@ -1,41 +1,60 @@
-body {
-  margin-top: 60px;
-}
+$white: #fff;
+$light-gray-white: #efefef;
+$dark-gray: #333;
+$medium-gray: #666;
+
 
-code {
-  background: #3b3f45;
-  color: #e0e0d7;
+.mainContainer {
+  padding: 40px 0;
+  transition: margin-left .5s;
+  width: 100%;
 }
 
-.card {
-  background: #fdfdfd;
-  border-top: solid 8px #bfbfbf;
-  box-shadow: 0 0 0 rgba(0, 0, 0, .25);
-  margin: 1.5em;
+.navZoolander-container {
+  height: 100%;
+  left: 0;
+  overflow-x: hidden;
+  position: fixed;
+  top: 0;
+  transition: .5s;
+  width: 0;
+  z-index: 1;
+}
 
-  .blue {
-    border-top-color: #739eaf;
-  }
+@media screen and (max-height: 450px) {
```

Comment:
Would it make sense to define max height as a variable since it's a breakpoint of sorts? Just a question, not a recommendation